package.json

```
@@ -42,7 +42,7 @@
   "devDependencies": {
     "@types/console-log-level": "1.4.2",
     "@types/node": "18.11.17",
-    "tempy": "1.0.1",
+    "tempy": "3.0.0",
```

```suggestion
    "tempy": "3.0.0",
```

This is esm-only &mdash; do not merge